conbench/api/results.py

```diff
@@ -302,6 +306,19 @@ def post(self) -> f.Response:
     view_func=benchmark_entity_view,
     methods=["GET", "DELETE", "PUT"],
 )
+
+# Towards the more explicit route path naming":
+# https://github.com/conbench/conbench/issues/972
+rule(
+    "/benchmark-results/",
+    view_func=benchmark_list_view,
+    methods=["GET", "POST"],
+)
```

Comment:
Created #1130 to implement this in client libraries.

Also, should these be plural? They always have been, but most of them operate on a single result. I tend to type it wrong when I guess.

Reply:
Great question! Lovely philosophical rabbit hole discussion: https://stackoverflow.com/questions/6845772/should-i-use-singular-or-plural-name-convention-for-rest-resources :)

But I think I've grown a preference for plural over the years :shrug: